doc/main.dox

```diff
@@ -122,7 +122,6 @@ The best way to learn how to use YARP is to read the tutorials and documentation
 \li <a href="http://playerstage.sourceforge.net/">Player/Stage</a>
 \li <a href="http://www.orocos.org/">Orocos</a>
 \li <a href="http://www.ros.org">ROS</a>
-\li <a href="http://gostai.com/products/urbi/">Urbi</a>
```

Comment:
Maybe we should also reorder them, ROS seems to me a bit more important than orocos and Player/stage nowadays